docs/modules/operation/pages/events/perf-data.adoc

```diff
@@ -0,0 +1,168 @@
+    </event>
+</events>
+```
+The example trap for the first event will be look like following.
+
+`snmptrap -v1 -c public 127.0.0.1:10162 .1.3.6.1.4.1.22222.2.4.3.12.2.2 '127.0.0.1' 6 4 100 .1.3.6.1.4.1.22222.2.4.3.12.2.21 s "Time=200 Value=300 Status=secondary Tag=Text"`
+
+You will notice in varbind that there is a named regex. In the collection, you must make sure the name is matching with the event's parameter name. And the value must be numeric. If the value is a string, you can use a paramValue tag to create a key-value mapping to translate the string into a numeric value. When you are using generic type time series, you will also want to register instance parameter. It will become part of the name of the time series. (Just like system mount point, it is dynamically created based on input.)
+P.S. It is important to quote the regex in CDATA.
+
+### create snmp-graph.properties
+You need to create a separate snmp-graph settings in etc/snmp-graph.properties.d directory
+
+Example:
+```
+reports=TIME, VALUE, STATUS, uei.opennms.org_traps_test_varbind
+
+report.TIME.name=TIME
+report.TIME.columns=TIME
+report.TIME.type=nodeSnmp
+report.TIME.command=--title="TIME" \
+ --vertical-label="Bytes per second" \
+ DEF:octIn={rrd1}:bytesIn:AVERAGE \
+ AREA:octIn#73d216: \
+ LINE1:octIn#4e9a06:"In " \
+ GPRINT:octIn:AVERAGE:"Avg  \\: %8.2lf %s" \
+ GPRINT:octIn:MIN:"Min  \\: %8.2lf %s" \
+ GPRINT:octIn:MAX:"Max  \\: %8.2lf %s\\n"
+
+report.STATUS.name=STATUS
+report.STATUS.columns=STATUS
+report.STATUS.type=nodeSnmp
+report.STATUS.command=--title="STATUS" \
+ --vertical-label="Bytes per second" \
+ DEF:octIn={rrd1}:bytesIn:AVERAGE \
+ AREA:octIn#73d216: \
+ LINE1:octIn#4e9a06:"In " \
+ GPRINT:octIn:AVERAGE:"Avg  \\: %8.2lf %s" \
+ GPRINT:octIn:MIN:"Min  \\: %8.2lf %s" \
+ GPRINT:octIn:MAX:"Max  \\: %8.2lf %s\\n"
+
+report.VALUE.name=VALUE
+report.VALUE.columns=VALUE
+report.VALUE.type=eventType
+report.VALUE.command=--title="VALUE" \
+ --vertical-label="Bytes per second" \
+ DEF:octOut={rrd1}:bytesOut:AVERAGE \
+ AREA:octOut#73d216: \
+ LINE1:octOut#4e9a06:"Out " \
+ GPRINT:octOut:AVERAGE:"Avg  \\: %8.2lf %s" \
+
+report.uei.opennms.org_traps_test_varbind.name=uei.opennms.org_traps_test_varbind
+report.uei.opennms.org_traps_test_varbind.columns=uei.opennms.org_traps_test_varbind
+report.uei.opennms.org_traps_test_varbind.type=interfaceSnmp
+report.uei.opennms.org_traps_test_varbind.command=--title="uei.opennms.org_traps_test_varbind" \
+ --vertical-label="Bytes per second" \
+ DEF:octIn={rrd1}:bytesIn:AVERAGE \
+ AREA:octIn#73d216: \
+ LINE1:octIn#4e9a06:"In " \
+ GPRINT:octIn:AVERAGE:"Avg  \\: %8.2lf %s"
+```
+
+#### create resource-types if you are using generic type
+If your time series data type is non nodeSnmp / interfaceSnmp. You will need to create a resource type file in etc/resource-types.d.
+The most important thing is to match the name with evenconf's resourceType. There is also ${instance} variable available.
```

Comment:
```suggestion
Make sure you match the name with evenconf's resourceType. 
There is also ${instance} variable available.
```
why is the {instance} variable important?

Reply:
for example you are receiving a network switch traps. {instance} can be port name. Without a proper name, you won't know what is that events bind to.